runtime/dancebox/tests/common/mod.rs

```diff
@@ -231,12 +232,13 @@ pub struct ExtBuilder {
     collators: Vec<(AccountId, Balance)>,
     // sudo key
     sudo: Option<AccountId>,
-    // list of registered para ids: para_id, genesis_data, boot_nodes, block_credits
+    // list of registered para ids: para_id, genesis_data, boot_nodes, block_credits, session_credits
     para_ids: Vec<(
         u32,
         ContainerChainGenesisData<MaxLengthTokenSymbol>,
         Vec<Vec<u8>>,
         u32,
+        u32,
```

Comment:
This should also be a struct, but don't change it in this PR.

Reply:
should not be too hard to change tbh

Reply:
Done in  [4e983c0](https://github.com/moondance-labs/tanssi/pull/401/commits/4e983c0798ad377cd020273c666936a4dc888caf)